lib/semian.rb

```diff
@@ -162,7 +167,7 @@ def resources
     Semian.logger.info("Semian sysv semaphores are not supported on #{RUBY_PLATFORM} - all operations will no-op")
   end
 
-  unless Semian.semaphores_enabled?
+  if ENV['SEMIAN_SEMAPHORES_DISABLED']
```

Comment:
This was done a few commits ago, when Simon suggested `#semaphores_enabled?` should include `&& Semian.sysv_semaphores_supported?`, which means the line here needs to be changed.